torch/optim/swa_utils.py

```diff
@@ -269,7 +269,7 @@ def update_bn(loader, model, device=None):
 
         model(input)
 
-    for bn_module in momenta.keys():
+    for bn_module in momenta:
```

Comment:
Would it be better to use an items() call here since we access the key in the next line?